src/hsmm_mvpy/models.py

```diff
@@ -1923,8 +2177,8 @@ def fit(self, step=1, verbose=True, end=None, trace=False, fix_iter=False, max_i
                 The maximum number of samples to explore within each trial
          	 trace: bool 
                 If True keep the scale and magnitudes parameters for each iteration
-         	 fix_iter: bool 
-                If True fix magnitudes and scale parameters for the previously found events
+         	 fix_prev: bool 
```

Comment:
This is no longer needed right? Because there is no way to fix previous iteration right? I'm fine with that, was a bit unreliable anyway

Reply:
Yes, it should go, it didn't work well.